redpanda/lib/testcontainers/redpanda.rb

```diff
@@ -75,8 +75,8 @@ def _startup_script
       startup_script = StringIO.new
       startup_script.print("#!/bin/sh\n")
       startup_script.print("/usr/bin/rpk redpanda start --mode dev-container")
```

Comment:
Just and improvement to avoid consuming more resources than expected

```suggestion
      startup_script.print("/usr/bin/rpk redpanda start --mode=dev-container --smp=1 --memory=1G")
```